src/lib/errors/errors.js

```diff
@@ -68,6 +68,12 @@ export class EmailNotConfirmedError extends ApolloError {
   }
 }
 
+export class InvalidEmailError extends Error {
+  constructor() {
+    super("Invalid email address. Please check the address and try again");
```

Comment:
```suggestion
    super("Invalid email address. Please use valid email address");
```
wdyt ?
cc @paolaperaza